pkg/apis/eksctl.io/v1alpha5/cloudwatch.go

```diff
@@ -8,6 +8,9 @@ type ClusterCloudWatch struct {
 
 // ClusterCloudWatchLogging container config parameters related to cluster logging
 type ClusterCloudWatchLogging struct {
+
+	// Types of logging to enable (see [CloudWatch docs](/usage/cloudwatch-cluster-logging/#clusterconfig-examples)).
+	// To choose from: api, audit, authenticator, controllerManager, scheduler
```

Comment:
We can also use `` formatting, e.g. https://master--eksctl.netlify.app/usage/schema/#nodeGroups-kubeletExtraConfig